android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -232,61 +218,38 @@ class RNPlayerView(
         addView(subtitleView)
     }
 
-    /**
-     * Called whenever this view's activity configuration changes.
-     */
-    override fun onConfigurationChanged(newConfig: Configuration?) {
-        super.onConfigurationChanged(newConfig)
-        pictureInPictureHandler?.onConfigurationChanged(newConfig)
-    }
-
-    /**
-     * Called when the player has just entered PiP mode.
-     */
-    override fun onEnterPictureInPicture() {
-        // Nothing to do
-    }
-
-    /**
-     * Called when the player has just exited PiP mode.
-     */
-    override fun onExitPictureInPicture() {
-        // Explicitly call `exitPictureInPicture()` on PlayerView when exiting PiP state, otherwise
-        // the `PictureInPictureExit` event won't get dispatched.
-        playerView?.exitPictureInPicture()
+    private fun isInPictureInPictureMode(): Boolean {
```

Comment:
This function has a very similar name to the property `isPictureInPictureMode` which makes it quite confusing. Should we rename to e.g. `isCurrentActivityInPictureInPictureMode ` or convert it to an extension function on the `activity`?

Reply:
https://github.com/bitmovin/bitmovin-player-react-native/pull/424/commits/d6e1247212145546f65b7f5bbc8b3ea54568bcc1